scripts/nettraffic

```diff
@@ -1,27 +1,43 @@
 #!/bin/sh
 
+# span for font
+fspan(){
+    echo "<span font_desc=\"${VALUE_FONT}\" color=\"${VALUE_COLOR}\"> ${1}</span>"
+}
+
+# span for label/icon
+lspan(){
+    echo "<span color=\"${LABEL_COLOR}\"> ${1}</span>"
+}
+
+# get font specifics from xresource file
 VALUE_COLOR=${color:-$(xrescat i3xrocks.value.color "#D8DEE9")}
 LABEL_COLOR=${label_color:-$(xrescat i3xrocks.label.color "#7B8394")}
-
 VALUE_FONT=${font:-$(xrescat i3xrocks.value.font "Source Code Pro Medium 13")}
 
+# read the dev file for extracting net usage
 GET="cat /proc/net/dev"
+
+# some default values for arguments"
 LABEL=true
```

Comment:
The "l" option also sets this to true.  Seems like either it should default to false or the flag should flip the value.